mmv1/templates/terraform/post_read/dns_policy_enable_dns64_all_queries.go.tmpl

```diff
@@ -0,0 +1,3 @@
+if err := d.Set("enable_dns64_all_queries", res["dns64Config"]["scope"]["allQueries"]); err != nil {
```

Comment:
res["dns64Config"]["scope"]["allQueries"]

I think we might be indexing into this object through an unsafe pattern.